pkg/controller/servicemesh/controlplane/reconciler.go

```diff
@@ -336,7 +337,7 @@ func mergeValues(base map[string]interface{}, input map[string]interface{}) map[
 	return base
 }
 
-func (r *ControlPlaneReconciler) getSMCPTemplate(name string) (v1.ControlPlaneSpec, error) {
+func (r *ControlPlaneReconciler) getSMCPTemplate(name string, maistraVersion string) (v1.ControlPlaneSpec, error) {
```

Comment:
In addition to finding the right charts, we will also have to change the default `tag` in the `base` template to the current version (e.g. `1.1.0`) - maybe we should start doing that programmatically, ie by merging it with a code-defined root template at reconcile time that just sets the `global.tag` to the current version, taken from `pkg/version`

Reply:
See my comment below on versioning, but we're only supporting the specification of major.minor within the SMCP spec.  ..z versions would require the use of a specific operator.  All we're doing with our multi-version support is adding support for the previous latest major.minor release, along with the current latest major.minor release (i.e. 1.0.4 and 1.1.0).

Maybe 'maistraVersion' is too specific, implying a full version, when it really only identifies a major.minor release.

Having said all of that, things like 'global.tag' are still defaulted in the default template/profile, just as they were previously.